### PR TITLE
Add browser compat for HTML Text content elements

### PR DESCRIPTION
**Pull request by teoli2003**
Add browser compat info for:
&lt;blockquote&gt;
&lt;dd&gt;
&lt;div&gt;
&lt;dl&gt;
&lt;dt&gt;
&lt;figcaption&gt;
&lt;figure&gt;
&lt;hr&gt;
&lt;li&gt;
&lt;main&gt;
&lt;ol&gt;
&lt;p&gt;
&lt;pre&gt;
&lt;ul&gt;

### CODE REVIEW

**Comment by teoli2003** [comment]
@Elchi3 PR rebased with new schema.

File: html/elements/li.json [hunk 1]
```diff
@@ -0,0 +1,156 @@
+{
+  "html": {
+    "elements": {
+      "div": {
```
Comment (Elchi3) [inline html/elements/li.json]: div -> li

File: html/elements/hr.json [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+                "version_added": true
+              },
+              "safari_ios": {
+                "version_added": true
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
```
Comment (Elchi3) [inline html/elements/hr.json]: I think hr align is deprecated.

File: html/elements/ol.json [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+                "version_added": true
+              },
+              "safari_ios": {
+                "version_added": true
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
```
Comment (Elchi3) [inline html/elements/ol.json]: I think `type` is deprecated.

File: html/elements/ul.json [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+                "version_added": true
+              },
+              "safari_ios": {
+                "version_added": true
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
```
Comment (Elchi3) [inline html/elements/ul.json]: I think `type` is deprecated.

File: html/elements/li.json (hunk 1 above)
Comment (teoli2003) [reply html/elements/li.json]: Fixed.

File: html/elements/hr.json (hunk 2 above)
Comment (teoli2003) [reply html/elements/hr.json]: Fixed

File: html/elements/ol.json (hunk 3 above)
Comment (teoli2003) [reply html/elements/ol.json]: Fixed.

File: html/elements/ul.json (hunk 4 above)
Comment (teoli2003) [reply html/elements/ul.json]: Fixed

**Comment by teoli2003** [comment]
@Elchi3 All review comments addressed (and trivial rebased done)

**Pull request merged**